### PR TITLE
Simplified the use of transChoice()

### LINKED ISSUE #7172: Add docs for the transChoice simplification
In https://github.com/symfony/symfony/pull/19795 @bocharsky-bw introduced a nice simplification when using `transChoice()`. I propose to update the current docs to use this new way and add a note about passing a explicit parameter for more complex use cases.

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7172.

I'm not happy with the result, so I'm open to comments to improve this. Thanks!

### CODE REVIEW

**Comment by bocharsky-bw** [comment]
Sounds good for me 👍

File: components/translation/usage.rst [hunk 1]
```diff
@@ -255,14 +255,27 @@ To translate pluralized messages, use the
...
-        10,
-        array('%count%' => 10)
+        10
     );
 
 The second argument (``10`` in this example) is the *number* of objects being
 described and is used to determine which translation to use and also to populate
 the ``%count%`` placeholder.
 
+.. note::
```
Comment (stof) [inline components/translation/usage.rst]: shouldn't this use a ``..versionadded`` directive ?

File: components/translation/usage.rst [hunk 2]
```diff
@@ -255,14 +255,27 @@ To translate pluralized messages, use the
 
     $translator->transChoice(
         'There is one apple|There are %count% apples',
-        10,
-        array('%count%' => 10)
```
Comment (stof) [inline components/translation/usage.rst]: do you still have any example passing other placeholders to transchoice ?

Comment (HeahDude) [inline components/translation/usage.rst]: `Using the translator` page has a first section about this (http://symfony.com/doc/2.7/components/translation/usage.html#message-placeholders).

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**